pkgs/tools/admin/chkservice/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, cmake, ninja, pkg-config, systemd, ncurses, lib }:
```

Comment:
is there a consensus about how to format this? I often see it formatted with one item per line instead of one single line.

Reply:
Technical answer: nixpkgs-format accepts it.
```
nixpkgs-fmt  pkgs/tools/admin/chkservice/default.nix
0 / 1 have been reformatted
```

I think the idea of nixpkgs-format is that you must be consistent in newlines. As it all fits on  one line, nixpkgs-fmt is happy.

nixpkgs-fmt does not check for very long lines. But this one is short enough that it still makes sense.

Reply:
In general I think it's a good idea to have multiple lines since it makes future edits easier and git diffs better. For such a leaf package that never gets updated (except treewide changes) it probably doesn't matter at all though.